src/Aperture/elliptical.jl

```diff
@@ -0,0 +1,89 @@
+export EllipticalAperture,
+       EllipticalAnnulus
+
+"""
+   EllipticalAperture(x, y, a, b, theta)
+
+A elliptical aperture.
+
+Where a >= b > 0 and -pi/2 <= theta <= pi/2
+
+# Examples
+```jldoctest
+julia> EllipticalAperture(2,2,2,2,1.2)
+EllipticalAperture(2.0, 2.0, a=2.0, b=2.0, theta=1.2)
+
+```
+"""
+struct EllipticalAperture{T <: Number} <: AbstractAperture
+    x::T
+    y::T
+    a::T
+    b::T
+    theta::T
+    cxx::T
+    cxy::T
+    cyy::T
```

Comment:
Are these quantities fundamental and used enough to store inside the struct? Or does it make more sense to call some function like
```
cxx, cyy, czz = coefficients(::EllipticalAperture)
```

I currently don't see these being used, so I wouldn't keep them in the struct.